signingscript/test/helpers/task_generator.py

```diff
@@ -0,0 +1,34 @@
+import os
+import json
+
+
+class TaskGenerator(object):
+    def __init__(self, urls=None):
+        self.urls = urls if urls is not None else [
+            'https://queue.taskcluster.net/v1/task/ZB9X1cvCRo-hTps7BDk8pw/artifacts/\
+public%2Fbuild%2Ffirefox-52.0a1.en-US.win64.installer.exe'
+        ]
+
+    def generate_file(self, work_dir):
```

Comment:
This could easily be a standalone function:

The extremely generic function would be `def dump_json(data, path):`

A specialized function might be `def dump_task_json(work_dir, data=None):` which calls `generate_json()` if data is None.